apps/docs/src/guide/getting-started/react-example.md

```diff
@@ -0,0 +1,35 @@
+# React Example
+
+<!-- TODO: Create properly code snippet on new package: `app/react-app` after https://github.com/FuelLabs/fuels-ts/pull/827 got merged -->
+
+```tsx
+import { BN, Provider, Wallet } from "fuels";
+import { useEffect, useState } from "react";
+
+function App() {
+  const [balance, setBalance] = useState(0);
+
+  useEffect(() => {
+    const main = async () => {
+      const provider = await Provider.create(
+        "https://mainnet.fuel.network/v1/graphql",
+      );
+
+      const wallet = Wallet.fromAddress("0x...", provider);
+      const { balances } = await wallet.getBalances();
+
+      setBalance(new BN(balances[0].amount).toNumber());
+    };
+
+    main();
+  }, []);
+
+  return <div>My Balance: {balance}</div>;
+}
+
+export default App;
```

Comment:
Do code snippets work for demo apps?

Reply:
I haven't tested it, but probably not with the new snippet format with the wrapping, etc. Maybe we can still import a snippet from elsewhere.

Reply:
- https://github.com/FuelLabs/fuels-ts/issues/3512